.github/workflows/build-release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build Release
+
+on: [ push, pull_request ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+  PROTOC: protoc
+
+jobs:
+  build_release:
+    name: Build Release
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      matrix:
+        config:
+          - { os: ubuntu-latest, toolchain: stable, target: x86_64-unknown-linux-gnu, cross: false }
+          - { os: ubuntu-latest, toolchain: stable, target: aarch64-unknown-linux-gnu, cross: true }
+          - { os: macos-11, toolchain: stable, target: x86_64-apple-darwin, cross: false }
+          - { os: macos-11, toolchain: stable, target: aarch64-apple-darwin, cross: false }
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: ${{ matrix.config.target }}
+
+      - uses: Swatinem/rust-cache@v1
+
+      - name: Build Databend
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          use-cross: ${{ matrix.config.cross }}
+          args: --release --target ${{ matrix.config.target }}
+        env:
+          CARGO_INCREMENTAL: '0'
+
+      - name: Build perf-tools
+        uses: actions-rs/cargo@v1
+        # Only build perf tools on ubuntu
+        if: "contains(matrix.config.os, 'ubuntu-latest')"
+        with:
+          command: build
+          use-cross: ${{ matrix.config.cross }}
+          args: --release --target ${{ matrix.config.target }} --bin databend-benchmark
+        env:
+          CARGO_INCREMENTAL: '0'
+          RUSTFLAGS: "-C link-arg=-Wl,--compress-debug-sections=zlib-gabi"
+
+      - if: "contains(matrix.config.os, 'ubuntu-latest')"
```

Comment:
cross: false

Reply:
We already use RUSTFLAGS, so objcopy can be removed

Reply:
Addressed.